patch-package
--- a/node_modules/coincap-lib/src/fetch.js
+++ b/node_modules/coincap-lib/src/fetch.js
@@ -1,6 +1,9 @@
-const fetch = require('isomorphic-fetch')
+'use strict';
 
-const { baseUrl } = require('../config')
+var fetch = require('isomorphic-fetch');
+
+var _require = require('../config'),
+    baseUrl = _require.baseUrl;
 
 /**
  * Executes an HTTP GET request to the CoinCap JSON API.
@@ -8,8 +11,12 @@ const { baseUrl } = require('../config')
  * @param  {String} path is the endpoint path.
  * @return {Promise}     a promise that will resolve to a JSON object.
  */
-function fetchApi (path) {
-  return fetch(`${baseUrl}${path}`).then(res => res.json())
+
+
+function fetchApi(path) {
+  return fetch('' + baseUrl + path).then(function (res) {
+    return res.json();
+  });
 }
 
-module.exports = fetchApi
+module.exports = fetchApi;
--- a/node_modules/coincap-lib/src/index.js
+++ b/node_modules/coincap-lib/src/index.js
@@ -1,60 +1,70 @@
-const io = require('socket.io-client')
+'use strict';
 
-const fetch = require('./fetch')
+var io = require('socket.io-client');
 
-const { baseUrl } = require('../config')
+var fetch = require('./fetch');
+
+var _require = require('../config'),
+    baseUrl = _require.baseUrl;
 
 /**
  * Constructs the CoinCap API object.
  *
  * @return {Object}
  */
-function createCoincap () {
-  const socket = io(baseUrl, { autoConnect: false })
 
-  const api = {}
+
+function createCoincap() {
+  var socket = io(baseUrl, { autoConnect: false });
+
+  var api = {}
 
   // Add JSON API supported endpoints
-  ;[
-    {
-      method: 'coins',
-      url: () => '/coins'
-    },
-    {
-      method: 'map',
-      url: () => '/map'
-    },
-    {
-      method: 'front',
-      url: () => '/front'
-    },
-    {
-      method: 'global',
-      url: () => '/global'
-    },
-    {
-      method: 'coin',
-      url: coin => `/page/${coin}`
-    },
-    {
-      method: 'coinHistory',
-      url: (coin, days) => `/history/${days ? `${days}day/` : ''}${coin}`
+  ;[{
+    method: 'coins',
+    url: function url() {
+      return '/coins';
+    }
+  }, {
+    method: 'map',
+    url: function url() {
+      return '/map';
+    }
+  }, {
+    method: 'front',
+    url: function url() {
+      return '/front';
     }
-  ].forEach(function ({ method, url }) {
-    api[method] = (...args) => fetch(url(...args))
+  }, {
+    method: 'global',
+    url: function url() {
+      return '/global';
+    }
+  }, {
+    method: 'coin',
+    url: function url(coin) {
+      return '/page/' + coin;
+    }
+  }, {
+    method: 'coinHistory',
+    url: function url(coin, days) {
+      return '/history/' + (days ? days + 'day/' : '') + coin;
+    }
+  }].forEach(function (_ref) {
+    var method = _ref.method,
+        url = _ref.url;
+
+    api[method] = function () {
+      return fetch(url.apply(undefined, arguments));
+    };
   })
 
   // Add Socket.IO methods
-  ;[
-    'open',
-    'close',
-    'on',
-    'off'
-  ].forEach(function (method) {
-    api[method] = socket[method].bind(socket)
-  })
+  ;['open', 'close', 'on', 'off'].forEach(function (method) {
+    api[method] = socket[method].bind(socket);
+  });
 
-  return api
+  return api;
 }
 
-module.exports = createCoincap()
+module.exports = createCoincap();
